<?xml version="1.0"?>

<launch>
  <node name="sbg_ellipse" pkg="sbg_driver" type="ellipse" output="screen">
  	<rosparam command="load" file="$(find kart_2dnav)/params/ellipse_N_default.yaml" />
  </node>
</launch>
